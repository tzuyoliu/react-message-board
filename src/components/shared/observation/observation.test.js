import React from 'react'
import { create } from 'react-test-renderer'
import Observation from '~/components/shared/observation'

describe('observation.test.js', () => {
    test('<Observation /> should render', () => {
        const observation = create(<Observation />)
        expect(observation).toBeTruthy()
    })

    test('<Observation /> should match the snapshot', () => {
        const observation = create(<Observation>Snapshot Test</Observation>)
        expect(observation.toJSON()).toMatchSnapshot()
    })
})
