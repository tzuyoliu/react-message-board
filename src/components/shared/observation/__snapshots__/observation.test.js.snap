// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`observation.test.js <Observation /> should match the snapshot 1`] = `
<span
  className="sc-bdVaJa bzOsUm"
>
  Snapshot Test
</span>
`;
